plugins/mac-videotoolbox/encoder.c

```diff
@@ -459,46 +457,40 @@ static void dump_encoder_info(struct vt_encoder *enc)
 								 : "default");
 }
 
-static void vt_video_info(void *data, struct video_scale_info *info)
+static bool set_video_format(struct vt_encoder *enc, enum video_format format,
+			     enum video_range_type range)
 {
-	struct vt_encoder *enc = data;
-
-	if (info->format == VIDEO_FORMAT_I420) {
-		enc->obs_pix_fmt = info->format;
+	bool fullrange = range == VIDEO_RANGE_FULL;
```

Comment:
Minor nitpick, but I'm not sure if the proper snake-case variant would be `full_range` here.

Reply:
It's definitely not less correct